admin/acceptance.py

```diff
@@ -641,9 +643,10 @@ def _provision_node(self, reactor, node):
         """
         commands = node.provision(package_source=self.package_source,
                                   variants=self.variants)
-        if self.dataset_backend == backends.ZFS:
-            zfs_commands = configure_zfs(node, variants=self.variants)
-            commands = commands.on(success=lambda _: zfs_commands)
+        commands = commands.on(
+            success=lambda _:
```

Comment:
below you added the node=node scoping trick, but here you did not. Is there a reason why it's needed below and not here?